_schema.graphql

```diff
@@ -317,6 +333,50 @@ type AnalyticsPartnerStats {
     @deprecated(reason: "Use audience() { uniqueVisitors } instead")
 }
 
+# Publish artwork Series Stats
+type AnalyticsPartnerStatsArtworksPublished {
+  partnerId: String!
+  percentageChanged: Int!
+  period: AnalyticsQueryPeriodEnum!
+
+  # Partner artworks published count time series
+  timeSeries(
+    cumulative: Boolean = false
+  ): [AnalyticsPartnerStatsArtworksPublishedTimeSeries!]!
+  totalCount: Int!
+}
+
+# Artworks published time series data of a partner
+type AnalyticsPartnerStatsArtworksPublishedTimeSeries {
+  count: Int
+  endTime: AnalyticsDateTime
+  startTime: AnalyticsDateTime
+}
+
+# Stats for pageviews of partner content
+type AnalyticsPartnerStatsPageviews {
+  artworkViews: Int!
+  galleryViews: Int!
+  partnerId: String!
+  percentageChanged: Int!
+  period: AnalyticsQueryPeriodEnum!
+  showViews: Int!
+
+  # Pageviews time series
+  timeSeries(
+    cumulative: Boolean = false
+  ): [AnalyticsPartnerStatsPageviewsTimeSeries!]
+  totalCount: Int!
+  uniqueVisitors: Int!
+}
+
+# Pageviews time series data of a partner
+type AnalyticsPartnerStatsPageviewsTimeSeries {
```

Comment:
Does this mean these fields haven't been accessible to any client yet?

I wonder if these additions/depreciations going unnoticed means Volt is still expecting to use the deprecated fields? Like `PageviewStats` for example.

Reply:
yeah i'm not sure what's going on here actually, it could be a change in the stitching code. Would be good to confirm.

Reply:
On second read and a bit more thought, I think it's ok for Volt to continue using these for now. This could be another #future-friday initiative, or we could even fold it in with converting Volt's use of artemis to relay.

Reply:
Ah didnt see your second comment there ^^
cool! ok so they dont block requests/raise errors?

Reply:
yeah exactly adding a `deprecation_reason` is graphql-ruby's way of marking a field deprecated, which only affects tools and DX, like introspection queries _(where it'll be grouped under "deprecated fields" or something)._ But deprecated fields are still able to be queried.

Link to the deprecation section of the latest version of the spec: https://spec.graphql.org/June2018/#sec-Field-Deprecation

Reply:
Thanks for the spec!